website/content/docs/services/discovery/dns-configuration.mdx

```diff
@@ -16,7 +16,7 @@ The Consul DNS is the primary interface for querying records when Consul service
 By default, the Consul DNS listens for queries at `127.0.0.1:8600` and uses the `consul` domain. Specify the following parameters in the agent configuration to determine DNS behavior when querying services:
 
 - [`client_addr`](/consul/docs/agent/config/config-files#client_addr)
-- [`ports.dns`](/consul/docs/agent/config/config-files#dns_port)
+- [`ports.dns`](/consul/docs/agent/config/config-files#dns_port) : Port `53`, which is typically reserved for the default port for DNS resolvers, is not used by default because it requires an escalated privilege to bind to. 
```

Comment:
so we done bury the lede

```suggestion
- [`ports.dns`](/consul/docs/agent/config/config-files#dns_port) : Consul does not use port `53`, which is typically reserved for the default port for DNS resolvers, by default because it requires an escalated privilege to bind to. 
```